### PR TITLE
Better printouts for stats

### PR DESCRIPTION
**Pull request by Jackojc**
- Clearer stats
- Better token printing
- remove benchmarking from lexer.cpp, didnt make sense to have it in there.

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -77,38 +90,269 @@ int main(int, char const*[]) {
...
+
+        ws::module::successln("File  => ", fmt_num_str(recv_time),  "ms");
+        ws::module::successln("Lexer => ", fmt_num_str(lexer_time), "ms");
+        ws::module::successln("Total => ", fmt_num_str(total_time), "ms");
+
+        ws::module::print_lines(1);
+        ws::module::noticeln("Input      => ", fmt_num_str(data.size()), " B");
+        ws::module::noticeln("Tokens     => ", fmt_num_str(tokens.size()));
+        ws::module::noticeln("Capacity   => ", fmt_num_str(tokens.capacity()));
+        ws::module::noticeln("Difference => ", fmt_num_str(tokens.capacity() - tokens.size()));
```
Comment (Ralakus) [inline main.cpp]: The `=>` is two spaces out of alignment

Comment (Jackojc) [reply main.cpp]: How do you mean? The intention was for them to line up in 2 columns.

Comment (Ralakus) [reply main.cpp]: Oh wait, Github is doing the weird font squishing thing

Comment (Jackojc) [reply main.cpp]: Ah yeah, no worries

File: callbacks.h [hunk 2]
```diff
@@ -512,9 +603,33 @@ void op_handler(StringIter& iter, Group& tokens) {
 
 
 
-void whitespace_handler(StringIter& iter, Group&) {
-    iter.next_while([] (auto&, char c) {
-        return (static_cast<bool>(std::isspace(c)));
+void whitespace_handler(StringIter& iter, Group& tokens) {
+    ws::module::run_if<lex::constant::PRINT_TOKENS>([&] {
```
Comment (Hazurl) [inline callbacks.h]: that's just `if constexpr (lex::constant::PRINT_TOKENS) { ... } else { ... }`

File: callbacks.h [hunk 3]
```diff
@@ -525,50 +640,58 @@ void whitespace_handler(StringIter& iter, Group&) {
 
 
 
-void left_brace_handler(StringIter&, Group& tokens) {
+void left_brace_handler(StringIter& iter, Group& tokens) {
     ws::module::run_if<lex::constant::PRINT_TOKENS>([&] {
```
Comment (Hazurl) [inline callbacks.h]: I just realized you use `run_if` everywhere, did I miss something or you forget you could use `if constexpr` ?

Comment (Jackojc) [reply callbacks.h]: Forgot tbh lol

Comment (Hazurl) [reply callbacks.h]: well time for a big commit then 👌

**Comment by Jackojc** [comment]
@Hazurl fixed

**Pull request merged**